Cocktails9/Cocktails9/TopRowView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  topRow.swift
+//  Cocktails9
+//
+//  Created by Mateja Mitic on 2.12.24..
+//
+
+import SwiftUI
+
+struct TopRowView: View {
+    @State private var searchQuery: String = ""  // For holding search text
+    @State private var isFiltering: Bool = false  // For handling filter state
+    
+    var body: some View {
+            HStack {
```

Comment:
I think you need to put this HStack in Navigation Bar. Here's a tutorial for that https://www.hackingwithswift.com/quick-start/swiftui/how-to-add-bar-items-to-a-navigation-view